core/src/com/unciv/ui/MultiplayerScreen.kt

```diff
@@ -353,12 +327,8 @@ class EditMultiplayerGameInfoScreen(game: GameInfo, gameName: String, backScreen
         rightSideButton.onClick {
             rightSideButton.setText("Saving...".tr())
             try {
-                backScreen.removeFromList(game.gameId)
-                //using addMultiplayerGame will download the game from Dropbox so the descriptionLabel displays the right things
-                backScreen.addMultiplayerGame(game.gameId, textField.text)
-                GameSaver().deleteSave(gameName, true)
-                UncivGame.Current.setScreen(backScreen)
-                backScreen.reloadGameListUI()
+                GameSaver().saveGame(game, textField.text, true)
+                UncivGame.Current.setScreen(MultiplayerScreen())
```

Comment:
One line from the UX improvements sneaked in here.
It isn't a huge problem but I could revert it because it may make saving from the edit screen slower.